html-templates/verified_credentials.html

```diff
@@ -285,11 +285,15 @@ <h5 v-if="state.showScanned" class="fw-bolder mb-3">
     </div>
   </script>
 
+  <script src="https://cdn.jsdelivr.net/npm/ua-parser-js/dist/ua-parser.min.js"></script>
   <script type="text/javascript">
     const { createApp, ref, reactive, computed, watch } = Vue;
-    /**
+   /**
      * @typedef  {"intro" |"verified" | "failed" | "pending"| "expired" |"abandoned"} AppStates
      */
+    // user agent detection
+    const uap = new UAParser();
+    console.log(uap.getResult())
```

Comment:
Nit.: if we leave a console.log statement (maybe as a `console.debug`) we may want to add some info about what this is - i.e.: "Detected user agent: browserUA"